package.json

```diff
@@ -83,11 +83,11 @@
 		"build-languages-if-enabled": "node -e \"(process.env.BUILD_TRANSLATION_CHUNKS === 'true' || process.env.ENABLE_FEATURES === 'use-translation-chunks') && process.exit(1)\" || yarn run build-languages",
 		"calypso-doctor": "./node_modules/.bin/calypso-doctor",
 		"clean": "yarn run clean:packages && yarn run clean:build && yarn run clean:public && yarn run clean:translations",
-		"clean:build": "npx rimraf build 'client/server/bundler/*.json' client/server/devdocs/search-index.js",
+		"clean:build": "rm -rf build 'client/server/bundler/*.json' client/server/devdocs/search-index.js",
 		"clean:packages": "yarn workspaces foreach --all --parallel --verbose --exclude 'wp-calypso' run clean",
-		"clean:public": "npx rimraf public",
-		"clean:translations": "npx rimraf build/strings calypso-strings.pot 'chunks-map.*.json'",
-		"distclean": "yarn run clean && npx rimraf '**/node_modules' '**/.cache' .yarn/install-state.gz",
+		"clean:public": "rm -rf public",
+		"clean:translations": "rm -rf build/strings calypso-strings.pot 'chunks-map.*.json'",
+		"distclean": "yarn run clean && rm -rf '**/node_modules' '**/.cache' .yarn/install-state.gz",
```

Comment:
It won't work because the quotes, I'll remove them.

I teststed it by adding `test-glob: "ls packages/**/package.json"`. It works as expected using all shells in my system (`/bin/ksh -c "yarn test-glob"`, `/bin/zsh -c "yarn test-glob"`, `/bin/bash -c "yarn test-glob"`). It makes sense because these scripts don't use Bash, they use `yarn shell` (https://yarnpkg.com/api/modules/yarnpkg_shell.html).

Reply:
TIL that Yarn implements a tiny shell and doesn't use the system shell at all. Only the binaries like `rm` or `cp` must be provided by the system.

And indeed, their command line parser (grammar [here](https://github.com/yarnpkg/berry/blob/master/packages/yarnpkg-parsers/sources/grammars/shell.pegjs)) only parses unquoted (plain) strings as "glob" AST nodes.